Utils/git_pull_master_into_fork.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+CONTENT_URL='https://github.com/demisto/content.git'
+
+if [ -z "$1" ]
+then
+  CURRENT=$(git branch --show-current)
+else
+  CURRENT=$1
+fi
+
+(
+  git remote add upstream_content $CONTENT_URL ||
+  git remote set-url upstream_content $CONTENT_URL
+) &&
+git fetch upstream_content &&
+git checkout master &&
+git rebase upstream_content/master &&
+git push -f origin master &&
```

Comment:
why do you force-pushing to master?

Reply:
I used the advice given [here](https://stackoverflow.com/questions/7244321/how-do-i-update-or-sync-a-forked-repository-on-github/7244456#7244456). This is relevant when the contributor already have some pushed commits in his branch (as I understand).

Reply:
add a comment at the top of the file that only contributors should run this script.